### ISSUE TITLE
Data Documentation

### ISSUE THREAD

**Issue by agrignard** [laatabix/I-Maroc#1]
In zonage_pdu would it be possible to know what is the following attributes:

- gen_move
- percent_TC
- move_TC

We also have the population from 2008 is there any chance to get something more updated?

**Comment by laatabix** [comment]
- I renamed 'gen_move' to 'total_move' : it represents the total recorded movements during a day, including pedestrians, cars, etc.
- 'percent_TC' represents the percentage of movements on the bus network.
- 'move_TC' is the number of trips on buses.

I added the label for each zone.
and included the 2023 population using an official projection of district population and the percentage of population in each zone relative to the district.

**Comment by agrignard** [comment]
Ok I great will integrate those change. What do you mean by the "percentage of population in each zone relative to the district" I am not sure that this attribute is present in the data? I see only pop2023

Also just to understand the data , the trips were collected in 2009 right? So it's more in link with the pop2008 I guess (we can stiil visulize the pop of 2023 in anycase)

**Comment by laatabix** [comment]
I used the 2008 population data for zones to calculate the percentage share of each zone. The projected 2023 population is provided by district, so I used those percentages to distribute the 2023 population among the zones... a very basic way to have an updated population estimate for the zones.

Trips were collected in 2008 (the final report was presented in 2009), so yes, I believe it is more appropriate to use pop2008.
